### PR TITLE
Licenses requirements update

### PR DESCRIPTION
**Pull request by joinimran**
I have updated license requirements for Microsoft Defender Advanced Threat Protection. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7094

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Microsoft Defender Advanced Threat Protection requires one of the following Micr
 
 > [!NOTE]
 > Eligible Licensed Users may use Microsoft Defender Advanced Threat Protection on up to five concurrent devices.
+> Microsoft Defender Advanced Threat Protection is also available for purchase from a Cloud Solution Provider (CSP). When purchased via CSP it does not require Microsoft Volume Licensing offers listed.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:45]:
```suggestion
> Microsoft Defender Advanced Threat Protection is also available for purchase from a Cloud Solution Provider (CSP). When purchased via a CSP, it does not require Microsoft Volume Licensing offers listed.
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ Microsoft Defender Advanced Threat Protection requires one of the following Micr
...
 
 
 Microsoft Defender Advanced Threat Protection is also available for purchase from a Cloud Solution Provider (CSP).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:48]:
```suggestion
```

Comment (JohanFreelancer9) [reply windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:48]: Duplication of the information in the note.

Comment (mjcaparas) [reply windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:48]: The second sentence is not approved by marketing and should not be included.

**Comment by joinimran** [comment]
Applied changes as per @JohanFreelancer9

**Pull request merged**